src/test/features/ccj/routes/paid-amount.ts

```diff
@@ -0,0 +1,93 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+
+import { Paths } from 'ccj/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+import { PaidAmountOption } from 'ccj/form/models/paidAmount'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const externalId = 'b17af4d2-273f-4999-9895-bce382fa24c8'
```

Comment:
externalId is available on the sampleClaimObj no need to hardcode, I see I messed it up in the their-details test can you fix?

Reply:
done